client/my-sites/site-settings/start-over/index.jsx

```diff
@@ -1,97 +1,80 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:my-sites:site-settings' ),
-	page = require( 'page' );
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-var HeaderCake = require( 'components/header-cake' ),
-	ActionPanel = require( 'my-sites/site-settings/action-panel' ),
-	ActionPanelTitle = require( 'my-sites/site-settings/action-panel/title' ),
-	ActionPanelBody = require( 'my-sites/site-settings/action-panel/body' ),
-	ActionPanelFigure = require( 'my-sites/site-settings/action-panel/figure' ),
-	ActionPanelFooter = require( 'my-sites/site-settings/action-panel/footer' ),
-	Button = require( 'components/button' ),
-	Gridicon = require( 'components/gridicon' ),
-	support = require( 'lib/url/support' );
+import HeaderCake from 'components/header-cake';
+import ActionPanel from 'my-sites/site-settings/action-panel';
+import ActionPanelTitle from 'my-sites/site-settings/action-panel/title';
+import ActionPanelBody from 'my-sites/site-settings/action-panel/body';
+import ActionPanelFigure from 'my-sites/site-settings/action-panel/figure';
+import ActionPanelFooter from 'my-sites/site-settings/action-panel/footer';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import support from 'lib/url/support';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
 
-module.exports = React.createClass( {
+const StartOver = ( { translate, selectedSiteSlug } ) => {
+	const strings = {
+		startOver: translate( 'Start Over' ),
+		contactSupport: translate( 'Contact Support' ),
+		emptySite: translate( 'Follow the Steps' )
+	};
```

Comment:
Only the `startOver` string is used more than once. I think it would be okay to just inline these and simplify the function